libs/contract/package.json

```diff
@@ -25,7 +25,9 @@
     "build:src": "run-p build:src:*",
     "build:src:cjs": "yarn tsc --project tsconfig.build.cjs.json",
     "build:src:mjs": "yarn tsc --project tsconfig.build.mjs.json",
-    "build:src:copy": "cpx \"./src/types/*.d.ts\" ./build/cjs/types && cpx \"./src/types/*.d.ts\" ./build/mjs/types",
+    "build:src:copy": "run-p build:src:copy:*",
+    "build:src:copy:types": "cpx \"./src/types/*.d.ts\" ./build/cjs/types && cpx \"./src/types/*.d.ts\" ./build/mjs/types",
+    "build:src:copy:deploy": "cpx ./src/deployed/** ./build/cjs/deployed && cpx ./src/deployed/** ./build/mjs/deployed",
```

Comment:
A build run managed to forget to copy deployment folder (?!?), so forcing it here.